shapeshift/web
# chore: enable gnosis, gnosis cowswap support

## Description

This PR enables Gnosis Chain support along with Cowswap support for Gnosis swaps

- turns gnosis base support on for all environments
- turns gnosis cowswap support on for all environments

## Pull Request Type


## Risk

Moderate

## Testing

- Verify gnosis base support works as expected
- Verify gnosis trading via Cowswap works as expected
- Verify gnosis trading via 1inch works as expected
- Verify Cowswap ETH swapping still works 


### Engineering

Relevant PR's:

https://github.com/shapeshift/web/pull/4482
https://github.com/shapeshift/web/pull/4583
https://github.com/shapeshift/web/pull/4595
https://github.com/shapeshift/web/pull/4618

### Operations

☝️ 

## Screenshots (if applicable)
